File: Utils/Assertions.java
package Utils_Compile; // rename as needed

import dafny.TypeDescriptor;

public class Assertions {

  static public <T> void assertEquals(dafny.TypeDescriptor<T> typeDescriptor, T left, T right) {
    org.junit.jupiter.api.Assertions.assertEquals(left, right);
  }

  static public <T> void expectEquals(dafny.TypeDescriptor<T> typeDescriptor, T left, T right) {
    org.junit.jupiter.api.Assertions.assertEquals(left, right);
  }

  static public <T> void assertNotEquals(dafny.TypeDescriptor<T> typeDescriptor, T left, T right) {
    org.junit.jupiter.api.Assertions.assertNotEquals(left, right);
  }

  static public <T> void expectNotEquals(dafny.TypeDescriptor<T> typeDescriptor, T left, T right) {
    org.junit.jupiter.api.Assertions.assertNotEquals(left, right);
  }

  static public void assertTrue(Boolean condition) {
    org.junit.jupiter.api.Assertions.assertTrue(condition);
  }

  static public void expectTrue(Boolean condition) {
    org.junit.jupiter.api.Assertions.assertTrue(condition);
  }

  static public void assertFalse(Boolean condition) {
    org.junit.jupiter.api.Assertions.assertFalse(condition);
  }

  static public void expectFalse(Boolean condition) {
    org.junit.jupiter.api.Assertions.assertFalse(condition);
  }
}

File: Utils/Assertions.cs
namespace Utils_Compile { // change as needed

  public partial class Assertions {
    public Assertions() {
    }

    public static void assertEquals<T>(T a, T b) {
      Xunit.Assert.Equal(a, b);
    }

    public static void expectEquals<T>(T a, T b) {
      Xunit.Assert.Equal(a, b);
    }

    public static void assertNotEquals<T>(T a, T b) {
      Xunit.Assert.NotEqual(a, b);
    }

    public static void expectNotEquals<T>(T a, T b) {
      Xunit.Assert.NotEqual(a, b);
    }

    public static void assertTrue(bool condition) {
      Xunit.Assert.True(condition);
    }

    public static void expectTrue(bool condition) {
      Xunit.Assert.True(condition);
    }

    public static void assertFalse(bool condition) {
      Xunit.Assert.False(condition);
    }

    public static void expectFalse(bool condition) {
      Xunit.Assert.False(condition);
    }
  }
}